run-qemu/run.sh

```diff
@@ -114,6 +114,15 @@ then
         "${GITHUB_ACTION_PATH}/print_test_summary.py" -s "${GITHUB_STEP_SUMMARY}" -j "${KERNEL_TEST}.json"
     fi
 fi
+
+# Try to collect $IMG:/command_output to OUTPUT_DIR
+if [[ -n ${OUTPUT_DIR} ]]
+then
+    mkdir -p ${OUTPUT_DIR}
+    guestfish --ro -a "$IMG" -i tar-out /command_output/ - \
```

Comment:
Would you mind using long options when available? Thanks.

Reply:
Done